docs/guides/security-policy-for-authors.md

```diff
@@ -519,13 +527,13 @@ The following CPAN modules may be useful:
 
 ## License and use of this document
 
-* Version: 1.4.0
+* Version: 1.4.1
 * License: [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/deed)
 * Copyright: © Robert Rothenberg <rrwo@cpan.org>, Some rights reserved.
 
 You may use, modify and share this file under the terms of the [CC-BY-SA-4.0](https://creativecommons.org/licenses/by-sa/4.0/deed) license.
 
-The actual security policies written based on this guide, including any recommended wording or example policies, are in the [public domain](https://creativecommons.org/public-domain/cc0/).
+The actual security policies written based on this guide, including any recommended wording or example policies, are in the public domain as per the [Zero-Clause BSD License](https://opensource.org/license/0bsd).
```

Comment:
Could we add a few words in the final commit message (for posterity) on why `0bsd` was preferred to `cc0`?